docs/.gitignore

```diff
@@ -21,3 +21,7 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+/docs/developers/contracts/references/aztec-nr
+
+/src/preprocess/AztecnrReferenceAutogenStructure.json
```

Comment:
Just a note that `yarn start:dev:local` didn't work immediately. I had to run run `yarn build` first, because of this error:

![image](https://github.com/AztecProtocol/aztec-packages/assets/18372439/0e40e96b-a80f-457d-b22b-5a5b6437f038)

Perhaps committing `AztecnrReferenceAutogenStructure.json` would resolve this?

Reply:
just changed the script